rtos/tests/TESTS/mbed_rtos/heap_and_stack/CMakeLists.txt

```
@@ -1,18 +1,20 @@
-# Copyright (c) 2020 ARM Limited. All rights reserved.
+# Copyright (c) 2020-2021 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+include(mbed_greentea)
 
-set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../../.. CACHE INTERNAL "")
-set(TEST_TARGET mbed-rtos-heap-and-stack)
+if(DEFINED TARGET_CORTEX_A)
```

`TARGET_CORTEX_A` is not a CMake variable.

```suggestion
if("CORTEX_A" IN_LIST MBED_TARGET_LABELS)